trillian/db/init.sql

```diff
@@ -0,0 +1,164 @@
+-- ---------------------------------------------
+-- Tree stuff here
+-- ---------------------------------------------
+
+-- Enable strict mode, so invalid data on inserts/updates is treated as error
+-- instead of warning.
+-- https://dev.mysql.com/doc/refman/5.7/en/sql-mode.html#sql-mode-strict
+SET GLOBAL sql_mode = 'STRICT_ALL_TABLES';
+
+CREATE DATABASE trillian;
```

Comment:
By the way, this hardcodes the DB name - not sure if there's a better alternative.